cerberus/validator.py

```diff
@@ -281,7 +287,7 @@ def _error(self, *args):
                 if rule == 'nullable':
                     constraint = field_definitions.get(rule, False)
                 else:
-                    constraint = field_definitions[rule]
+                    constraint = field_definitions.get(rule)
```

Comment:
is this a special case for `required`? i'd rather keep that prone to a `KeyError` exception than have possibly incorrect `None` values in the data.

Reply:
Is it correct?
``` python
if rule == 'nullable':
    constraint = field_definitions.get(rule, False)
elif rule == 'required':
    constraint = field_definitions.get(rule, self.require_all)
else:
    constraint = field_definitions[rule]
```

Reply:
looks good.